### PR TITLE
[INLONG-5245][Manager] Optimize the URI paths of the WebAPI and OpenAPI

### LINKED ISSUE #5245: [Improve][Manager] Optimize the URI paths of the WebAPI and OpenAPI
### Description

There are two groups of manager controllers. The open api group url starts with "api/inlong/manager/openapi" and the web group starts with "api/inlong/manager" which is cumbersome and inconvenient.

Change them to **/manager/openapi** and **/manager/api** respectively.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by woofyzhao**
- Fixes #5245

There are two groups of manager controllers. The open api group url starts with "api/inlong/manager/openapi" and the web group starts with "api/inlong/manager" which is cumbersome and inconvenient.

Change them to /manager/openapi and /manager/api respectively.

### CODE REVIEW

File: inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/fetcher/ManagerFetcher.java [hunk 1]
```diff
@@ -154,7 +154,7 @@ private String buildBaseUrl() {
     /**
      * build vip url for manager according to config
      *
-     * example - http://127.0.0.1:8080/api/inlong/manager/openapi/agent/getInLongManagerIp
+     * example - http://127.0.0.1:8080/manager/openapi/agent/getInLongManagerIp
```
Comment (healchow) [inline inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/fetcher/ManagerFetcher.java:157]: The `getInLongManagerIp` already changed, please change to `getManagerIpList`, thanks.

File: inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/WebBaseTest.java [hunk 2]
```diff
@@ -57,11 +57,10 @@
 public abstract class WebBaseTest extends BaseTest {
 
     public MockMvc mockMvc;
-
-    @Resource
-    private WebApplicationContext context;
     @Resource
     ObjectMapper objectMapper;
```
Comment (healchow) [inline inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/WebBaseTest.java:61]: This variable should be `private`.

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by pocozh — APPROVED** [review]
LGTM

**Comment by pocozh** [comment]
Would it be better to add `inlong` in the url?

**Review by kipshi — APPROVED** [review]

**Comment by woofyzhao** [comment]
> Would it be better to add `inlong` in the url?

OK

**Pull request merged**